style.css

```diff
@@ -0,0 +1,123 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: 375px;
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 50px;
+  background: #3c3a39;
+}
+
+.logo {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  display: flex;
+  position: relative;
+  width: 77px;
+  height: 24px;
+  left: 12px;
+  top: 0%;
+  color: white;
+}
+
+.menu-btn {
+  position: relative;
+  top: 10%;
+  right: 30px;
+}
+
+.menu {
+  width: 35px;
+  height: 5px;
+  background-color: white;
+  margin: 6px 0;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  background: #1c1a19;
+  top: 50px;
+  left: 0;
+  background-image: url("background.png");
+  background-repeat: no-repeat;
+  background-position: right 0 bottom -50px;
+  background-size: 35%;
+}
```

Comment:
- [ ] Kindly make sure your design matches the Figma design
Your design
<img width="370" alt="Screen Shot 2022-03-18 at 06 44 21" src="https://user-images.githubusercontent.com/28307960/158939112-0b8d4d72-2d44-4ea5-b41b-09a8c7d04720.png">
Figma design
<img width="350" alt="Screen Shot 2022-03-17 at 03 20 38" src="https://user-images.githubusercontent.com/28307960/158939145-517b2302-29e4-41a5-8831-72979b70ce24.png">